CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - [usd#1961](https://github.com/Autodesk/arnold-usd/issues/1961) - Random curves width in Hydra when radius primvars are authored
 - [usd#1977](https://github.com/Autodesk/arnold-usd/issues/1977) - Aov shaders not set properly in hydra mode of the scene format
 - [usd#1984](https://github.com/Autodesk/arnold-usd/issues/1984) - Cylinder lights not taking normalization into account through USD
+- [usd#1992](https://github.com/Autodesk/arnold-usd/issues/1992) - Hydra skinned positions cannot have more than 3 keys
```

Comment:
I suggest to rephrase this line, as when reading it I am not sure if it is a fix or if it is forcing the position to have 3 keys or less. May be something like "Allow hydra skinned positions to have more than 3 keys" ?